tests/unit/controllers/start/hello-test.js

```diff
@@ -10,8 +10,28 @@ moduleFor('controller:start/hello', 'Unit | Controller | start/hello', {
   ]
 });
 
-// Replace this with your real tests.
 test('it exists', function(assert) {
+  assert.expect(3);
   let controller = this.subject();
-  assert.ok(controller);
+
+  controller.uploadingImage = true;
```

Comment:
Shouldn't we use `set(controller, 'uploadingImage', true);` here?

Same for the rest of the code, really.